### ISSUE TITLE
Premiered date localization

### ISSUE THREAD

**Issue by CGarces** [Varstahl/xbmc#6]
I'm using the addon with one spanish account.

The premiered information is not correct due issues with the locale.

This ugly hard-code just works for me.

  ```
#25 de agosto de 2017
locale.setlocale(locale.LC_TIME, "es_ES.utf8")
if (None is not res[4]): meta['videometa']['premiered'] = datetime.datetime.strptime(res[4], '%d de %B de %Y').strftime('%Y-%m-%d')

```

Maybe the locale information can be fetched from `<script id="av-wconf-dv-web-player-cfg" type="application/json">`

**Comment by Varstahl** [comment]
Thanks for the snippet, will come in handy. A lot of information will be in need of i18n and l10n, dates are no difference. As soon as I'll start localization I'll make sure to remember about this.

**Comment by CGarces** [comment]
If you point me to some code (from another addon) maybe I can help with the  i18n and l10n.

**Comment by Varstahl** [comment]
I'd love to help you but I wouldn't know where to start either. While I'm half experienced with python I only ever managed those problems with php or c/c++. I'm really hacking around with kodi at the moment, but seems to be working fine (_streams are finally working, yay!_ :D). The fix doesn't seem too hard though, but I have to learn a bit about localization in python and kodi, and also check if they work in both linux and win.

Btw, if you're not familiar with the terms, i18n stands for internationalization (coding in a way that supports translation and localization of content) and l10n stands for localization (adapting various text and numbers to adapt to the human's language rulesets, like translations, decimal points or date formats).

For general reference, in case you're interested, [gettext](https://docs.python.org/3/library/gettext.html) (link specific to python) is the usual way to go, but kodi seems to use a slightly different system I have to look into.

**Comment by Varstahl** [comment]
I was just fiddling with this to implement it in the release, I just realized your code won't work in several scenarios. That code only work in Linux, and on top of that only if the locale is installed. Need to do it the stupid way I think, there's no way around it.

**Comment by Varstahl** [comment]
Addressed in d463340ad59c5198a6532332c1a0929096d6e67e
